src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -133,9 +129,21 @@ class AutoColumnSize extends BasePlugin {
      * @type {Boolean}
      */
     this.inProgress = false;
+    /**
+     * Number of already measured columns (we already know their sizes).
+     */
+    this.measuredColumns = 0;
```

Comment:
Missing JSDoc tag and if the property is public should be added to the TS definition.